docs/modules/release-notes/pages/2023/v-3-26-legacy-jan-07-2023.adoc

```diff
@@ -0,0 +1,29 @@
+= v3.26| Legacy January 7, 2023
```

Comment:
Ensure the page title and nav title match exactly.

Also be sure there is a space between `v3.26` and the `|` symbol, like this: 

`v3.26 | Legacy January 7, 2023`